### PR TITLE
Connect docs

### PR DESCRIPTION
**Pull request by belvitadream**

### CODE REVIEW

File: docs/.algolia/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ALGOLIA_APPLICATION_ID=L5GQNBSXIK
+ALGOLIA_WRITE_API_KEY=ec1733d4be602661319a240958f17ffb
+ALGOLIA_SITE_URL=https://connect.nightly.app/docs/
+ALGOLIA_INDEX_NAME=NightlyConnect
+ALGOLIA_SEARCH_API_KEY=57f60c8df985aec6f8f44287f6b98c16
```
Comment (NorbertBodziony) [inline docs/.algolia/.env:5]: We should move it to secrets right ?

File: docs/docs/client/connect.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+```js
+import { ClientSui } from '@nightlylabs/nightly-connect-sui'
+
+const client: ClientSui = await ClientSui.create({
+  url: RELAY_ENDPOINT // default: https://nc2.nightly.app
+})
+const info: GetInfoResponse = await client.getInfo(sessionId)
+
+const msg: Connect = {
+  publicKeys: ['1', '2'],
```
Comment (NorbertBodziony) [inline docs/docs/client/connect.md:73]: maybe use real values + good names

File: docs/docs/client/push.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: Push notifications
+slug: client/push
+---
+
+Push server will allow to applications notify the user on incoming requests for previously established connections.  
+Push notifications are being implemented for the client, when the device is locked. Though when the user application is open and session is active, all incoming requests are added to the List of PendingRequests. Once a client opens an application on its devices, he will have the option to decide on the transaction (approve or reject it).
```
Comment (NorbertBodziony) [inline docs/docs/client/push.md:7]: its more about calling defined endpoint with POST call you should also show here example + maybe link to it

**Review by NorbertBodziony — CHANGES_REQUESTED** [review]

File: docs/docs/client/connect.md (hunk 2 above)
Comment (belvitadream) [reply docs/docs/client/connect.md:73]: done

**Pull request merged**